integration-tests/bats/blame-system-view.bats

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+    setup_repository
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+setup_repository() {
```

Comment:
These are great tests, but going forward we want bats tests of SQL features to mostly be smoke tests. Most of these would be better expressed as engine tests.

Reply:
Cool; happy to move these to engine tests.

Reply:
Actually... how do you feel about leaving the BATS test around for this initial commit to get this view out to customers faster, then I can loop back to converting them to enginetests? Having a DOLT_CONFIG() function to change the user for the blame test data would make writing the enginetests cleaner. Happy to push them into enginetests now if you feel strongly about it though.

Reply:
Fine for now, sure